src/Input/Silk.NET.Input.Sdl/SdlKeyboard.cs

```diff
@@ -23,6 +23,8 @@ public SdlKeyboard(SdlInputContext ctx)
         public event Action<IKeyboard, Key, int>? KeyDown;
         public event Action<IKeyboard, Key, int>? KeyUp;
         public event Action<IKeyboard, char>? KeyChar;
+        public void BeginInput() => _ctx.Sdl.StartTextInput();
+        public void EndInput() => _ctx.Sdl.StopTextInput();
```

Comment:
Isn't this going to be inconsistent with GLFW? For example on windows I believe this may open the on-screen keyboard in some cases.

Reply:
This will not open the on screen keyboard on windows.